src/SelectPanel/SelectPanel.tsx

```diff
@@ -61,6 +60,7 @@ export function SelectPanel({
   textInputProps,
   overlayProps,
   sx,
+  renderFn,
```

Comment:
Noticed this is a new addition to the API, but was not able to understand from the stories/examples when it should be used 🤔 

Can you explain this prop?

Reply:
I removed this as a prop and just have a function defined in FilteredActionList that handles this. If we want to expose this as a customizable prop later, we can, but it currently covers every use case I saw.

Reply:
That's perfect! thanks!